.github/workflows/main.yml

```diff
@@ -26,6 +27,7 @@ jobs:
       with:
         node-version: ${{ steps.nvm.outputs.nvmrc }}
     - run: npm ci
+      if: steps.cache-node.outputs.cache-hit != 'true'
```

Comment:
Does it make that much of a difference running `npm ci` if there's a cache hit?

Reply:
Not a big difference, but still it download all manifests and rebuilds binaries when needed.